src/App.jsx

```diff
@@ -1,58 +1,30 @@
 import { useEffect, useState } from "react";
 import "./App.css";
 import Navbar from "./components/Navbar";
-import axios from "axios";
-import Articles from "./components/Articles";
+
+import Articles from "./components/articles";
```

Comment:
Articles needs to be spelt with a capital "A" - the same as the component filename.